client/Client.cpp

```diff
@@ -593,7 +598,8 @@ void CClient::battleStarted(const BattleInfo * info)
 
 	if(info->tacticDistance && vstd::contains(battleints, info->sides[info->tacticsSide].color))
 	{
-		boost::thread(&CClient::commenceTacticPhaseForInt, this, battleints[info->sides[info->tacticsSide].color]);
+		PlayerColor color = info->sides[info->tacticsSide].color;

```

Comment:
can be moved out: 
`if(info->tacticDistance && vstd::contains(battleints, info->sides[info->tacticsSide].color))`
replace with
```
PlayerColor color = info->sides[info->tacticsSide].color;
if(info->tacticDistance && vstd::contains(battleints, color))
```

Reply:
There are many `color` inside that function (`battleStarted`), moving the `color` out might confuse other people in the future, I think there is a reason to keep this inside the if bracket.